src/WorldStorage/WSSAnvil.cpp

```diff
@@ -3039,6 +3055,64 @@ void cWSSAnvil::LoadWolfOwner(cWolf & a_Wolf, const cParsedNBT & a_NBT, int a_Ta
 
 
 
+void cWSSAnvil::LoadOcelotOwner(cOcelot & a_Ocelot, const cParsedNBT & a_NBT, int a_TagIdx)
+{
+	// Load the owner information. OwnerUUID or Owner may be specified, possibly both:
+	AString OwnerUUID, OwnerName;
+	int OwnerUUIDIdx = a_NBT.FindChildByName(a_TagIdx, "OwnerUUID");
+	if (OwnerUUIDIdx > 0)
+	{
+		OwnerUUID = a_NBT.GetString(OwnerUUIDIdx);
+	}
+	int OwnerIdx = a_NBT.FindChildByName(a_TagIdx, "Owner");
+	if (OwnerIdx > 0)
+	{
+		OwnerName = a_NBT.GetString(OwnerIdx);
+	}
+	if (OwnerName.empty() && OwnerUUID.empty())
+	{
+		// There is no owner, bail out:
+		return;
+	}
+
+	// Convert name to UUID, if needed:
+	if (OwnerUUID.empty())
+	{
+		// This wolf has only playername stored (pre-1.7.6), look up the UUID
+		// The lookup is blocking, but we're running in a separate thread, so it's ok
+		OwnerUUID = cRoot::Get()->GetMojangAPI().GetUUIDFromPlayerName(OwnerName);
```

Comment:
I don't think this is a good idea. Imagine loading a vanilla world with cuberite that has a dozens of ocelots with owners :)